examples/energy-monitor/ui/widgets/navigation.slint

```diff
@@ -6,7 +6,8 @@ import { FloatButton } from "float_button.slint";
 import { Theme } from "../theme.slint";
 import { Images } from "../images.slint";
 
-export component Navigation {
+export component Navigation inherits Rectangle {
```

Comment:
Any reason to do that instead of using `preferred-width: 100%;  preferred-height: 100%;` ?

Reply:
Rectangle was only for testing I removed it now.